ckanext/data_qld/reporting/helpers/helpers.py

```diff
@@ -4,11 +4,17 @@
 import pytz
 from datetime import datetime, timedelta
 
+import ckantoolkit as tk
```

Comment:
Why was this moved up? Also, it doesn't need two blank lines after it.

Reply:
+

Reply:
Will be fixed and pushed by Alex shortly.